user_docs/en/userGuide.t2t

```diff
@@ -2089,6 +2089,21 @@ The tone will increase in pitch every space, and for a tab, it will increase in
 - Both Speech and Tones: This option reads indentation using both of the above methods.
 -
 
++++ Document Navigation +++[DocumentNavigation]
+This category allows you to adjust various aspects of document navigation.
+
+==== Paragraph Style ====[DocumentFormattingParagraphStyle]

```

Comment:
Please include the definition list for the setting: https://github.com/nvaccess/nvda/blob/master/devDocs/userGuideStandards.md#feature-settings

```suggestion
==== Paragraph Style ====[DocumentFormattingParagraphStyle]
: Default (Handled by application)
  Enabled
: Options
  Default (Handled by application), Handled by application, Normal, Block
:
```